src/jsMain/kotlin/PayloadJS.kt

```diff
@@ -9,3 +9,8 @@ data class SignUpResponseJS(
     val UserConfirmed: Boolean = false,
     val UserSub: String
 )
+
+data class MfaSettingJS(
```

Comment:
Why do you need JS classes for MfaSetting,  AssociateSoftwareTokenResponse and VerifySoftwareTokenResponse? They don't seem to be incompatible with js/ts, or am I missing something?